addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -123,9 +123,9 @@ Wysiwyg.include({
                 const $codeview = $('<textarea class="o_codeview d-none"/>');
                 self.$editable.addClass('o_editable oe_structure');
 
+                $iframeTarget.append($codeview);
```

Comment:
Is that change directly related to our mass mailing customisations?

Reply:
Yes. A bit further down I move the `#iframe_target` element so it's at the same place no matter the context and this impacts the display of the code view.